tai_mux/static_platform_adapter.cpp

```diff
@@ -0,0 +1,39 @@
+#include "static_platform_adapter.hpp"
+
+StaticPlatformAdapter::StaticPlatformAdapter(uint64_t flags, const tai_service_method_table_t* services) {
+    auto config_file = std::string(std::getenv(TAI_MUX_STATIC_CONFIG_FILE.c_str()));
```

Comment:
You might want to consider having a default location for the config file which is used in the case when the environment variable is not set.

Reply:
Do you think `/etc/tai/mux/static.json` is a reasonable place for it?

Reply:
Yes, that sounds like a good default location.